client/src/components/Explorer/ExplorerItem.js

```diff
@@ -47,15 +39,15 @@ const ExplorerItem = ({ item, onClick }) => {
         href={`${ADMIN_URLS.PAGES}${id}/edit/`}
         className="c-explorer__item__action c-explorer__item__action--small"
       >
-        {editIcon}
+        <Icon name="edit" title={`${STRINGS.EDIT} ${title}`} />
```

Comment:
I would expect this (and the button below) to instead use what @gasman suggested in https://github.com/wagtail/wagtail/pull/5377#issuecomment-502167009.

…so this particular line would become:

```suggestion
        <Icon name="edit" title={STRINGS.EDIT.replace('{0}', title)} />
```

And we would need the corresponding change to the translatable `EDIT` label (need to check whether it’s used elsewhere or not).

Otherwise translations won’t be correct for languages where the order of the sentence is different. I’d also suggest wrapping the title in quotes to make it more likely screen readers read it out as something separate.

Reply:
@thibaudcolas this should all be addressed in https://github.com/wagtail/wagtail/pull/5386/commits/f35e27cde4b22392909047784b8ad85ec7e5b8c1

Reply:
Thank you!